core/include/mbox.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    core_mbox Mailboxes
```

Comment:
When I hear "message box" I think of a pop-up notification.

![](https://i.stack.imgur.com/9btLs.png "")